prediction_market_agent_tooling/tools/tokens/cross_chain_bridge.py

```
@@ -0,0 +1,231 @@
+        raise ValueError(
+            f"Bridge amount ${amount_usd:.2f} is below minimum ${BRIDGE_MIN_AMOUNT_USD}. "
+            "Small bridges are uneconomical due to gas costs."
+        )
+    
+    logger.info(
+        f"Initiating cross-chain bridge: {amount_wei.as_token} xDAI → {buy_token} on Polygon"
+    )
+    
+    try:
+        # Call Node.js bridge service
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)